README.md

```diff
@@ -604,6 +604,13 @@ dependencies {
 + Work was done to eliminate vulnerabilities in _common_ and _bom_ dependencies.
     + **th2-bom must be updated to _4.0.3_ or higher.**
 
+### 3.42.1
++ Added retry in case of a RabbitMQ channel or connection error (when possible).
++ Added InterruptedException to basicConsume method signature.
++ Added additional logging for RabbitMQ errors.
++ Fixed connection recovery delay time.
++ Integration tests for these scenarios.
```

Comment:
We merge into dev-version-5. These changes were not released for common v3. So, probably, we need to move correct the change log here